src/Login.js

```diff
@@ -0,0 +1,25 @@
+import { useContext } from "react";
+import { signInWithGoogle, signInAnon } from "./firebase";
+import { AuthContext } from "./Auth.js";
+import { Navigate } from 'react-router-dom';
+
+
+function Login() {
+
+    const { currentUser } = useContext(AuthContext);
+
+    if (currentUser) {
+        return <Navigate to="/saved" replace />;
+    }
+
+    return (
+        <div className="mainContent">
+            <div className='loginButtons'>
+                <button className="googleButton" onClick={signInWithGoogle}>Sign in with Google</button>
+                <button className="anonButton" onClick={signInAnon}>Sign in Anonymously</button>
```

Comment:
signInAnon works, but im wondering if it might be useful for a user to have context as to  what sign in anonymously does - like, how does one sign in anonymously? am i using a throwaway account that is auto generated? etc

Reply:
Added login description